src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetApprenticeship/GetApprenticeshipHandlerTests.cs

```diff
@@ -194,6 +194,7 @@ public void VerifyResultMapping()
                 Assert.AreEqual(Apprenticeship.FirstName, _result.FirstName);
                 Assert.AreEqual(Apprenticeship.LastName, _result.LastName);
                 Assert.AreEqual(Apprenticeship.Uln, _result.Uln);
+                Assert.AreEqual(Apprenticeship.NiNumber, _result.NINumber);
```

Comment:
No point in returning this information from Commitments as it is always null

Reply:
removed